src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/IS91DeterminationService.kt

```diff
@@ -20,6 +22,8 @@ class IS91DeterminationService(
         resultCodes.contains(it.secondaryResultCode) ||
         it.offenceCode == offenceCode
     }
+    log.info("All offence histories: ${offenceHistories.toString()}")
```

Comment:
meh.. for future reference, you don't need to explicitly call `toString`, its called by default when trying to print objects